uknetdev.c

```diff
@@ -384,11 +384,11 @@ static void uknetdev_updown(struct netif *nf)
 					("%s: Poll receive enabled\n",
 					 __func__));
 			/* Create a thread */
-			lwip_data->sched = uk_sched_get_default();
+			lwip_data->sched = uk_sched_current();
 			UK_ASSERT(lwip_data->sched);
 			lwip_data->poll_thread =
-				uk_sched_thread_create(lwip_data->sched, NULL,
-						       NULL, _poll_netif, nf);
+				uk_sched_thread_create(lwip_data->sched,
+						       _poll_netif, nf, NULL);
 #else /* CONFIG_HAVE_SCHED */
```

Comment:
Hi @skuenzer the PR looks good to me, I have tested this PR with virtio and https server and is running without any error.
I just have one question. Why `uk_sched_thread_create()` creates a thread without any stack ?

Reply:
Yes, this is true but `0x0` means that a stack is still allocated, just a default size is used instead.

`lib/uksched/include/uk/sched.h`:154:
```
 * @param stack_len
 *   Size of the thread stack. If set to 0, a default stack size is used
 *   for the stack allocation.
```